src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const kindOfAction of actions) {
```

Comment:
you don't need such complicated name, we can leave it as `action`

Reply:
well, I like it)))
but, if you think, it would be better to change them, there is no problem
Done 👌